WordPressKit/WordPressComRestApi.swift

```diff
@@ -612,6 +639,38 @@ open class WordPressComRestApi: NSObject {
             }
     }
 
+    public func upload(
+        _ URLString: String,
```

Comment:
Nitpick. Value names should start with lowercase letters.

```suggestion
        _ urlString: String,
```

Reply:
I have updated this part in https://github.com/wordpress-mobile/WordPressKit-iOS/pull/720/commits/a3c19a9e07d601e9341cabe150bdfa2b2bc24e1b. which probably made it worse, in terms of the name... I used `URLString` because that's the term used throughout this class...